crates/devolutions-pedm-shared/devolutions-pedm-client-http/src/models/error.rs

```diff
@@ -0,0 +1,47 @@
+/*
+ * Devolutions PEDM API
+ *
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 
+ * 
+ * Generated by: https://openapi-generator.tech
+ */
+
+use crate::models;
+use serde::{Deserialize, Serialize};
+
+/// 
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
+pub enum Error {
+    #[serde(rename = "AccessDenied")]
+    AccessDenied,
+    #[serde(rename = "NotFound")]
+    NotFound,
+    #[serde(rename = "InvalidParameter")]
+    InvalidParameter,
+    #[serde(rename = "InternalError")]
+    InternalError,
+    #[serde(rename = "Cancelled")]
+    Cancelled,
+
+}
+
+impl std::fmt::Display for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match self {
+            Self::AccessDenied => write!(f, "AccessDenied"),
+            Self::NotFound => write!(f, "NotFound"),
+            Self::InvalidParameter => write!(f, "InvalidParameter"),
+            Self::InternalError => write!(f, "InternalError"),
+            Self::Cancelled => write!(f, "Cancelled"),
+        }
```

Comment:
suggestion: If you only want to output the name of the variant instead of a pretty message, you can simply to this:
```suggestion
        write!(f, "{self:?}")
```